pandas/io/excel/_base.py

```diff
@@ -1474,6 +1474,10 @@ class ExcelFile:
             This is not supported, switch to using ``openpyxl`` instead.
     engine_kwargs : dict, optional
         Arbitrary keyword arguments passed to excel engine.
+
+    Examples
+    --------
+    >>> file = pd.ExcelFile('myfile.xlsx')  # doctest: +SKIP
```

Comment:
do we want to show it in an example of how it would be used? like
```python
   with pd.ExcelFile("myfile.xls") as xls:
       df1 = pd.read_excel(xls, "Sheet1")
```

Reply:
Done.